dbms/src/Common/ConcurrentIOQueue.h

```diff
@@ -26,132 +26,127 @@ namespace DB
   * while (queue.isFull()) {}
   * queue.nonBlockingPush(std::move(obj));
   * ```
-  * There are additional overheads compared to using mpmcqueue directly
-  * - two locks
-  * - condition double check
-  * Since io queues are not performance sensitive, they are acceptable.
   */
 template <typename T>
 class ConcurrentIOQueue
```

Comment:
`LooseBoundedMPMCQueue` is a better name in my opinion.

Reply:
good idea! renamed.